### PR TITLE
MINT-1989 Add a cache for wallets address

### PR DESCRIPTION
**Pull request by olegdizus**

### CODE REVIEW

**Comment by olegdizus** [comment]
> Oleg, could you please check the case, if user have two different Ledger devices?
> I suppose caches must kept separately for each device.

Fixed

File: packages/core/redux/wallets/actions.js [hunk 1]
```diff
@@ -78,73 +79,92 @@ export const initWallets = () => (dispatch) => {
...
+    [BLOCKCHAIN_DASH]: getDashSigner,
+    [BLOCKCHAIN_LITECOIN]: getLitecoinSigner,
+    [BLOCKCHAIN_NEM]: getNemSigner,
+    [BLOCKCHAIN_WAVES]: getWavesSigner,
+  }
+
+  Object.entries(signerSelectors).forEach(([ blockchain, signerSelector ]) => {
+    let address = addressCache[blockchain]
+    if(!address) {
+      const signer = signerSelector(state)
```
Comment (ozalexo) [inline packages/core/redux/wallets/actions.js]: await required here

File: packages/core/redux/wallets/actions.js [hunk 2]
```diff
@@ -78,73 +79,92 @@ export const initWallets = () => (dispatch) => {
...
+      if (signer) {
+        address = signer.getAddress(accountPath)
```
Comment (ozalexo) [inline packages/core/redux/wallets/actions.js]: We need specify correct accountPath for each blockchain. E.g. derivation path for BTC is different in mainnet/testnet

Also, accountPath decalred only at line 109 below.

**Comment by e-maximus** [comment]
Moved to https://github.com/ChronoBank/ChronoMint/pull/853

**Pull request closed without merging**